.pylintrc

```diff
@@ -9,7 +9,7 @@
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=compat.py
+ignore=compat.py, utils.py, path_resolver.py, validator.py
```

Comment:
curious, why are these added?

Reply:
I had to keep utils.py because of the copying of the which command, the other two can go away.